### PR TITLE
Mark console functions _static

### PR DESCRIPTION
**Pull request by Elchi3**
So this feels a little weird but it updates the console data according to the discussion in https://github.com/mdn/browser-compat-data/issues/20114.

### CODE REVIEW

File: api/_globals/console.json [hunk 1]
```diff
@@ -786,7 +796,8 @@
       },
       "groupEnd": {
```
Comment (saschanaz) [inline api/_globals/console.json:797]: (This should be `_static` too)

Comment (Elchi3) [reply api/_globals/console.json]: oops, fixed.

**Comment by saschanaz** [comment]
I have slight preference for things in namespaces not to be marked as static (as the IDL does not mark them as static either and thus the parser doesn't give the flag, which means any tool would need to check the type of the parent). But I guess not marking it is again problematic for MDN, so I guess either way is fine. 😅

**Review by queengooborg — APPROVED** [review]
Approving but not merging until the content changes have been made.

**Pull request merged**